src/Type/IntrospectionProvider.php

```diff
@@ -305,7 +305,7 @@ protected function registerIntrospectionTypes()
                                 'includeDeprecated' => ['type' => Boolean(), 'defaultValue' => false],
                             ],
                             'resolve' => function (TypeInterface $type, array $args): ?array {
-                                [$includeDeprecated] = $args;
+                                $includeDeprecated = $args['includeDeprecated'];
```

Comment:
I don't know if my previous comment is shown but because it is an key-value array.
`'includeDeprecated' => ['type' => Boolean(), 'defaultValue' => false],`

Reply:
In that case it should be `['includeDeprecated' => $includeDeprecated] = $args`. Shouldn't it?

Reply:
As long as you check that we don't get an undefined index error it should be fine.

Reply:
Just a question, is there any benefit from this syntax?